motor_micro_node:
  ros__parameters:
    motor_directions:
    - 1
    - 1
    - 1
    - 0
    - 1
    - 0
    - 1
    - 1
    motor_mappings:
    - 1
    - 7
    - 2
    - 3
    - 6
    - 4
    - 5
    - 8
